src/services/expenses.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+let expenses = [];
+
+const getExpenses = ({ userId, categories, from, to }) => {
+  if (userId) {
+    expenses = expenses.filter(expense => +expense.userId === +userId);
+  }
+
+  if (categories) {
+    expenses = expenses
+      .filter(expense => categories.includes(expense.category));
+  }
+
+  if (from) {
+    expenses = expenses.filter(expense => expense.spentAt >= from);
+  }
+
+  if (to) {
+    expenses = expenses.filter(expense => expense.spentAt <= to);
+  }
+
+  return expenses;
+};
+
+const getById = id => expenses.find(expense => +expense.id === +id);
```

Comment:
```suggestion
const getById = id => expenses.find(expense => Number(expense.id) === Number(id));
```

The same in other places